schemas/stsci.edu/asdf/0.1.0/transform/regions_selector.yaml

```diff
@@ -0,0 +1,55 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/transform/regions_selector"
+tag: "tag:stsci.edu:asdf/0.1.0/transform/regions_selector"
+title: >
+  Represents a RegionsSelector transform.
+description: |
+  A model which maps regions to their corresponding transforms.
+
+allOf:
+  - $ref: transform
+  - type: object
+    properties:
+      mask:
+        description: |
+          An array with the shape of the detector/observation.
+          Pixel values are of type int or str and represent
+          region labels as in the RegionSelector transform.
+          Pixels which are not within any region have value 0 or " ".
+
+        $ref: ./selector_mask
+      inputs:
+        description: |
+          Names of inputs.
+        type: array
+        items:
+          type: string
+      outputs:
+        description: |
+          Names of outputs.
+        type: array
+        items:
+          type: string
```

Comment:
I see.  Because this is so general, it doesn't have those names like most of the transforms do.  But are these input and output names overridable on any transform?  In which case, we should probably define inputs/outputs in the base class (`transform.yaml`) and handle them in the base class in the Python code as well.